Helper/WebHookHandlers/BillPaid.php

```diff
@@ -143,4 +146,4 @@ private function getOrder($incrementId)
 
         return false;
     }
-}
+}
```

Comment:
💅🏼 Para mantermos o padrão de termos uma linha em branco no final, o que acha?
```suggestion
}

```